server/utils/hollaex-tools-lib/tools/order.js

```diff
@@ -111,16 +116,28 @@ const getUserQuickTrade = async (spending_currency, spending_amount, receiving_a
 	let symbol = originalPair;
 	let side = 'sell';
 
-	// find if broker exists
-	let broker = await getModel('broker').findOne({ where: { symbol: originalPair } });
+	const QuickTradeModel = getModel('quickTrade');
+	
+	let quickTradeConfig = await QuickTradeModel.findOne({ where: { symbol: originalPair } });
```

Comment:
We could get the data from the init configuration to make it faster instead of querying the database @fetok12